chore: change the defaults of otelcol.exporter queue batch block (#5061)
grafana/alloy: docs/sources/shared/reference/components/otelcol-queue-batch-block.md

headless: true
---

Batching is disabled by default.
To enable it, explicitly include `batch {}` in your Alloy configuration.
You do not need to include a `batch {}` block in your `otelcol.exporter` if you already use a `otelcol.processor.batch` component,

> although batching in the exporter is the prefered method because it is more flexible.

Reviewer: ```suggestion
By default, batching is disabled .
To enable it, explicitly include `batch {}` in your {{< param "PRODUCT_NAME" >}} configuration.
You don't need to include a `batch {}` block in your `otelcol.exporter` if you already use a `otelcol.processor.batch` component, although batching in the exporter is the preferred method because it's more flexible.
```

The following arguments are supported:

| Name            | Type        | Description                                                                                                | Default      | Required |
| --------------- | ----------- | ---------------------------------------------------------------------------------------------------------- | ------------ | -------- |
| `flush_timeout` | `duration`  | Time after which a batch will be sent regardless of its size. Must be a non-zero value.                    |  `"200ms"`   | no       |
| `min_size`      | `number`    | The minimum size of a batch.                                                                               |  `2000`      | no       |
| `max_size`      | `number`    | The maximum size of a batch, enables batch splitting.                                                      |  `3000`      | no       |

> | `sizer`         | `string`    | How the queue and batching is measured. Overrides the sizer set at the `sending_queue` level for batching. |  `"items"`   | no       |

Reviewer: ```suggestion
| `sizer`         | `string`   | The method used to measure the queue and batching. Overrides the sizer set at the `sending_queue` level for batching. | `"items"` | no       |
```
